src/App.js

```diff
@@ -22,15 +28,15 @@ function App() {
     const fetchWeather = async () => {
       const message = query.q ? query.q : "current location.";
```

Comment:
if `message` variable is not in use anywhere else but `toast.info("Fetching weather for " + message);`, you should delete this const too so that there's no unused variables in your code

Reply:
> if `message` variable is not in use anywhere else but `toast.info("Fetching weather for " + message);`, you should delete this const too so that there's no unused variables in your code

Okay I will do that